requirements/base.txt

```diff
@@ -17,3 +17,5 @@ aws_lambda_builders==1.1.0
 tomlkit==0.5.8
 # TODO: remove six and its usage since we don't support Python 2
 six==1.15.0
+# pyinstaller should not update to 4.0 until https://github.com/pyinstaller/pyinstaller/issues/4674 is resolved
+pyinstaller==3.5
```

Comment:
`pyinstaller` is not a runtime dependency of SAM CLI. We should put this into dev.txt or a new file specifically for building the binary. If we were to leave it here, pyinstaller (and its depedencies) would be installed if someone still `pip install`'s, which is not what we want.